cli/src/main/java/org/eclipse/hono/cli/AbstractClient.java

```diff
@@ -30,21 +30,24 @@
  * A base class providing support for connecting to a Hono server.
  *
  */
-abstract class AbstractExampleClient {
+abstract class AbstractClient {
 
     protected final Logger LOG = LoggerFactory.getLogger(getClass());
 
     protected Context ctx;
     @Value(value = "${tenant.id}")
     protected String tenantId;
+    @Value(value = "${message.type:all}")
```

Comment:
as with the `tenant.id`, I'd rather have it set to the default value using the application.yml fiel (in which you already seem to set the default value).